### PR TITLE
Feature/cl 8956 add new partner api methods

### PR DESCRIPTION
**Pull request by chrisangelocloseapp**

### CODE REVIEW

**Comment by anton-close** [comment]
Seems good to me. Can you also add examples please (check example folder)?

**Comment by dpenasclose** [comment]
Shouldn't we also add the endpoints for the Message: Cards ([found in here](https://partner.closeapi.nl/api/documentation))?

With this we can add carousels and look for them, but we can't send images to them. There 4 methods that allows us to do that.

File: examples/carousel/createCarousel.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Sending a message to all users in all chats for one event.
+Do you want to send a message to all users going to a specific event? Then you can use the sendToAllChatsForEvent() operation.
```
Comment (dpenasclose) [inline examples/carousel/createCarousel.md:2]: Maybe this text should change? It is about creating the carousel, not sending a message. The one below (the lookup one) should also change.

**Review by anton-close — APPROVED** [review]

**Comment by anton-close** [comment]
after we merge this one, we have to a new release, so it's available in packagist if you download the package

**Pull request merged**